build.gradle.kts

```diff
@@ -214,11 +210,24 @@ subprojects {
             }
 
             withType(Javadoc::class) {
+                exclude("io/opentelemetry/internal/**")
```

Comment:
Should we exclude any internal directory not only the main one? If any subpackage has it's own internal for example?

Reply:
Good point, and since we shouldn't have `internal` in the package of any public class just expanded the glob a bit